src/components/todoBox.js

```diff
@@ -0,0 +1,76 @@
+import { v4 as uuidv4 } from 'uuid';
+import React from 'react';
+import TodosList from './todos';
+import Header from './header';
+import InputTodo from './input';
+
+class TodoContainer extends React.Component {
```

Comment:
- [ ] Here your filename is `todoBox.js` while the component it contains is `TodoContainer`. It would be nice to keep the component name and the file name similar to make the code easy to navigate. Thanks!